serverless.yml

```diff
@@ -84,3 +82,21 @@ functions:
       - schedule: cron(30 12,14,18,20 ? * MON,TUE,WED,THU,FRI *)
     tags:
       name: lambda_function_2-${self:custom.app_name}-${self:custom.customer}-production
+
+  alerts-eth:
+    environment:
+      CF_API_TOKEN: ${env:CF_API_TOKEN_2}
+      CF_CONTAINS_STRINGS: ${env:CF_CONTAINS_STRINGS}
+      CF_ZONE_NAMES: ${env:CF_ZONE_NAMES_2}
+      SES_AWS_REGION: ${env:AWS_REGION}
+      SES_SUBJECT: ${env:SES_SUBJECT}
+      SES_RETURN_TO_ADDR: ${env:SES_RETURN_TO_ADDR}
+      SES_RECIPIENT_EMAILS: ${env:SES_RECIPIENT_EMAILS}
+    handler: bin/alerts
+    timeout: 300
+    events:
+      # cron(Minutes Hours Day-of-month Month Day-of-week Year)
+      # Either `day-of-month` or `day-of-week` must be a question mark (?)
+      - schedule: cron(30 12,14,18,20 ? * MON,TUE,WED,THU,FRI *)
+    tags:
+      name: lambda_function_3-${self:custom.app_name}-${self:custom.customer}-production
```

Comment:
Not that it's a big deal, but it may come as a surprise later that the `..._2` env. vars are the ones being use for this `lambda_function_3-...` tagged function. It may be worth setting up unique env. vars for all three functions, even if the first and second functions use some of the same values for those env. vars. Just a thought.

Reply:
Good observation. I really want to redesign the configuration input for this, but it's kind of waiting on the side task of secrets and app configuration management.